starter_code/public/javascript/APIHandler.js

```diff
@@ -1,17 +1,52 @@
+/* jshint esversion:6 */
 class APIHandler {
   constructor (baseUrl) {
     this.BASE_URL = baseUrl;
   }
 
-  getFullList () {
+  getFullList() {
+    $.ajax({
+  url: this.BASE_URL + "/characters",
+  method: "GET",
+  success: function (response) {
+    console.log(response);
+
+  },
+  error: function (err) {
+    console.log(err);
+  },
+});
 
   }
 
   getOneRegister () {
+    $.ajax({
+  url: this.BASE_URL + "/characters/:id",
```

Comment:
Has tenido una pequeña confusión aquí: El :id se utiliza cuando estás definiendo una ruta, pero aquí no estamos definiendo rutas sino indicar una ruta de una api que vamos a utilizar para sacar información. Como lo que estamos haciendo es obtener información de un registro en concreto, efectivamente hay que pasarle un id, pero ese id es una variable. En código sería algo así.
getOneRegister(id){   <== Fijate que este id que le pasamos a la función por parámetro es el que se usa en la url
   $.ajax({
 +  url: this.BASE_URL + "/characters/" + id, <== Aquí.